agent/bench-scripts/postprocess/fio-postprocess-viz.py

```diff
@@ -41,12 +41,17 @@ def main(ctx):
     out_files[i].write("#LABEL:%s\n" % columns[i])
 
   with open(join(ctx.DIR, 'hist.csv'), 'r') as csv:
-    csv.readline()
+    while True:
+      l = csv.readline()
+      if l == None:
+          print('ERROR: hit end of file without seeing header')
+          sys.exit(1)
+      if l.startswith('msec'):
```

Comment:
Since we changed from milliseconds to seconds, why is the header still using `msec`?

Reply:
good point, but it doesn't really matter right?  Andrew said fio-postprocess-viz.py is going away to be replaced by browbench right?